llvm/lib/Passes/PassBuilder.cpp

```diff
@@ -947,6 +948,65 @@ parseLowerAllowCheckPassOptions(StringRef Params) {
   return Result;
 }
 
+Expected<MetaRenamerOptions> parseMetaRenamerPassOptions(StringRef Params) {
```

Comment:
We should unify the syntax to handle the case `pass-name<arg=a list>`, pass `internalize` has the same request but has a different syntax, pending on pass infrastructure maintainers.